packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -257,25 +247,22 @@ export const LocationDetailViewTable = (): JSX.Element => {
     type: 'LIST_LOCATION_ITEMS',
   });
 
-  const { prefix: initialPrefix } = location
-    ? parseLocationAccess(location)
-    : ({} as LocationData);
-
-  const prefix =
-    history.length === 1 ? initialPrefix : history[history.length - 1];
+  const prefix = history.join('');
```

Comment:
What does this do?

Reply:
I think with Caleb's latest changes, it updated how the entries in the navigation history is stored to just be specific folders, like `'folder1/` > `nestedFolder1/` for instance.

Before this change, we had entries in the history that looked like this: `public/folder1/` > `public/folder1/nestedFolder1/`

I think he needs to join it together in order to get the complete prefix again so that we can fetch the list items when we navigate back to the specific folder

Reply:
@timngyn Nailed it